src/construction.cpp

```diff
@@ -194,11 +195,12 @@ void construction_menu()
     draw_grid( w_con, w_list_width + w_list_x0 );
 
     //tabcount needs to be increased to add more categories
-    int tabcount = 9;
+    int tabcount = 10;
     std::string construct_cat[] = {_( "All" ), _( "Constructions" ), _( "Furniture" ),
```

Comment:
Not in this PR. If you wanted to try for a solution in a separate PR though I'm sure that would be appreciated